### ISSUE TITLE
Research | Enhanced Military Information | Stakeholder interview prep

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#65973]
## Background 

- [Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/military-information/2023-enhanced-military-information)
- [Research folder](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/military-information/discovery-and-research/2023-09-enhanced-military-info-mvp/stakeholder-interviews)

### What we know
- We have received a request from stakeholders to expose more military information for veterans ([8 key data elements, 6 to start while 2 are being discussed](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/military-information/2023-enhanced-military-information#what-can-we-do-instead)), based on the hypothesis that these data points will make it easier for them to apply for PACT act benefits.
- We know that legislation has been passed that requires us to give Veterans read access to their ILER, and a PDF download is an acceptable approach to meeting the requirements of that legislation. 

### Things we need to uncover
- We need a list of teams/forms who might be interested in taking advantage of using the data once it is available in the profile so we can socialize this with them
- TBD based on outcome of desk research

## Tasks 
- [x] Research plan is written ([draft started here](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/military-information/discovery-and-research/2023-09-enhanced-military-info-mvp/stakeholder-interviews/stakeholder-interview-research-plan.md))
- [x] Convo guide is written ([draft started here](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/identity-personalization/profile/military-information/discovery-and-research/2023-09-enhanced-military-info-mvp/stakeholder-interviews/stakeholder-interview-conversation-guide.md))
- [x] Identify stakeholders to interview
- [x] Schedule interviews

## Acceptance Criteria 
- [x] Ready to conduct stakeholder interviews

**Comment by mtcA6** [comment]
@pamela-drouin @andaleliz same w/this one could you add points. Thanks!

**Comment by andaleliz** [comment]
@mtcA6 thanks for starting this ticket!  I added an estimate (Pam, as always feel free to update), though I expect it to change based on the outcome of the desk research.

I'm going to update the description, but just want to make you aware of how I'm changing things in the spirit of clarity, and sharing about the research process:

>We know that most people access va.gov using a mobile device

The majority of visitors to VA.gov use desktop, rather than mobile devices. It's close to 50/50 though.

>how can we help the veteran understand the download/ what is in the download
>how do we make read access to the 8 key data elements useful to our users

These questions are better suited for conversations with users, not stakeholders.

**Comment by andaleliz** [comment]
@pamela-drouin FYI I started some draft docs to get a few things out of my head that I'm already thinking about. I added links to everything in the ticket description.  Contributions welcome and encouraged 😄

**Comment by andaleliz** [comment]
@pamela-drouin the docs are ready for your review! Here's the [research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/military-information/discovery-and-research/2023-09-enhanced-military-info-mvp/stakeholder-interviews/stakeholder-interview-research-plan.md), and [convo guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/military-information/discovery-and-research/2023-09-enhanced-military-info-mvp/stakeholder-interviews/stakeholder-interview-conversation-guide.md). Thanks in advance for your feedback.

**Comment by pamela-drouin** [comment]
@andaleliz really great work, Liz! My only feedback/question is around [Military data points](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/military-information/discovery-and-research/2023-09-enhanced-military-info-mvp/stakeholder-interviews/stakeholder-interview-conversation-guide.md#military-data-points) questions 2 and 3. They are so similar -- but I feel like you are trying to get at two different (yet related) points of clarity. Is there a way to make these questions more distinct?

**Comment by andaleliz** [comment]
@pamela-drouin thank you for pointing that out - good call! I clarified them to get at the two different things I'm hoping to dig into:

1. What do we already know about how the 8 data points identified are useful to Veterans?
2. Do we have insights into how Veterans are currently referencing this information when it's requested?

FYI, I've reached out to Melissa and she's looking at some dates early next week for our discussion. She wants to loop in Angela Allen who has knowledge about VADIR (VA Department of Defense Identity Repository) data that Melissa doesn't have.
